crates/router/src/lib.rs

```diff
@@ -20,10 +20,20 @@ use ya_sb_util::PrefixLookupBag;
 mod dispatcher;
 
 lazy_static! {
-    pub static ref GSB_PING_TIMEOUT: u64 = env::var("GSB_PING_TIMEOUT")
-        .unwrap_or("5".into())
+    static ref RAW_GSB_PING_TIMEOUT: u64 = env::var("GSB_PING_TIMEOUT")
+        .unwrap_or(20.to_string())
         .parse()
         .unwrap();
+    // After GSB_PING_TIMEOUT without reply the connection will disconnect ( default = 20s )
```

Comment:
default value within a comment will be outdated (not updated) soonish... pls remove it

Reply:
are we planning to change it again?

Will remove it :)